_includes/releases/v21.1/v21.1.16.md

```diff
@@ -0,0 +1,32 @@
+## v21.1.16
+
+Release Date: March 7, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v21-1-16-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused the query optimizer to omit join filters in rare cases when reordering joins, which could result in incorrect query results. This bug was present since v20.2. [#76621][#76621]
+- Fixed a bug where some rows within query results could be omitted if the query references a column within a composite key with a null value. Previously, CockroachDB could incorrectly omit a row from query results against a table with multiple column families when that row contains a NULL value when a composite type ([FLOAT](../v21.1/float.html), [DECIMAL](../v21.1/decimal.html), [COLLATED STRING](../v21.1/collate.html), or any arrays of such a type) is included in the [PRIMARY KEY](../v21.1/primary-key.html). For the bug to occur, a composite column from the PRIMARY KEY must be included in any column family other than the first one. [#76637][#76637]
+- Fixed a race condition that, in rare circumstances, could cause a node to panic with `unexpected Stopped processor` during shutdown. [#76828][#76828]
+- There is now a 1 hour timeout when sending Raft snapshots, to avoid stalled snapshot transfers. Stalled snapshot transfers could prevent Raft log truncation, thus growing the Raft log very large. This timeout is configurable via the `COCKROACH_RAFT_SEND_SNAPSHOT_TIMEOUT` environment variable. [#76830][#76830]
```

Comment:
nit that's just my opinion:
```suggestion
- There is now a 1 hour timeout when sending Raft snapshots to avoid stalled snapshot transfers. Stalled snapshot transfers could prevent Raft log truncation, thus growing the Raft log very large. This timeout is configurable via the `COCKROACH_RAFT_SEND_SNAPSHOT_TIMEOUT` environment variable. [#76830][#76830]
```

Reply:
Done. Thanks!